src/main/java/Kvothe.java

```diff
@@ -0,0 +1,204 @@
+        String description = "";
+        while (lastWordIndex < lineWords.length && !lineWords[lastWordIndex].startsWith("/")) {
+            description += lineWords[lastWordIndex] + " ";
+            lastWordIndex++;
+        }
+        newArgs[argsIndex++] = description.trim();
+        lastWordIndex++;
+
+        // Get values associated with specified arguments
+        for (int i = 0; i < args.length-1; i++) {
+            if (lastWordIndex >= lineWords.length) {
+                break;
+            }
+
+            String nextArg = "";
+            while (lastWordIndex < lineWords.length && !lineWords[lastWordIndex].startsWith("/")) {
+                nextArg += lineWords[lastWordIndex] + " ";
+                lastWordIndex++;
+            }
+
+            nextArg = nextArg.trim();
+            newArgs[argsIndex++] = nextArg;
+        }
+
+        //last argument
+        String lastArg = "";
+        lastWordIndex++;
+
+        while(lastWordIndex < lineWords.length) {
+            lastArg += lineWords[lastWordIndex++] + "";
+        }
+
+        newArgs[argsIndex] = lastArg.trim();
+
+        return newArgs;
+    }
+
+
+    private static void add(String line) {
+
+        String lineWords[] = line.split(" ");
+        String command = lineWords[0];
+        Task newTask = null;
+        String args[];
+
+        switch (command) {
+            case "todo":
+                String descr = line.substring("todo".length()).trim();
+                newTask = new Todo(descr);
+                break;
+            case "deadline":
+                args = parseLine(line, Deadline.args);
+                newTask = new Deadline(args[0], args[1]);
+                break;
+            case "event":
+                args = parseLine(line, Event.args);
+                newTask = new Event(args[0], args[1], args[2]);
+                break;
+            default:
+                echo("Sorry. I do not support that method.");
+                break;
+        }
+
+        tasks[tasksIndex] = newTask;
+        tasksIndex++;
+    }
+
+    private static void list() {
+        for (int i = 0; i < tasksIndex; i++) {
+            echo(i + 1 + ". " + tasks[i], i == 0, i == tasksIndex - 1);
+        }
+    }
+
+    private static void done(int index) {
+        tasks[index - 1].markAsDone();
+        echo("Nice! I've marked this task as done:\n\t\t" + tasks[index - 1]);
+    }
+
+    private static void mark(int index){
+        tasks[index - 1].setIsDone(true);
+        echo("Nice! I've marked this task as done:\n\t\t" + tasks[index - 1]);
+    }
+
+    private static void unmark(int index){
+        tasks[index - 1].setIsDone(false);
+        echo("OK, I've marked this task as not done yet:\n\t\t" + tasks[index - 1]);
+    }
+
+    private static int getTaskNumber(String[] args){
+
+        int index = 0;
+        if (args.length < 2 || args.length > 2) {
+           echo("Sorry. Please specify one task number.");
+           return -1;
```

Comment:
This seems like a magic number. Perhaps you could represent the -1 as a constant named INVALID_INDEX.